components/status/StatusActions.vue

```diff
@@ -55,7 +55,7 @@ function reply() {
 
     <div flex-1>
       <StatusActionButton
-        :content="$t('action.boost')"
+        :content="$t(!!status.reblogged ? 'action.boosted' : 'action.boost')"
```

Comment:
You can remove the `!!` from these conditions

Reply:
sure

Reply:
check ` :active="!!status.bookmarked"`